### PR TITLE
Update manager architecture

### PR DESCRIPTION
**Pull request by szpotona**
- metrics workflow, logs and events, workflow execution flow

### CODE REVIEW

File: content/manager_architecture/metrics-flow.md [hunk 1]
```diff
@@ -21,30 +21,40 @@ You can send metrics to the management environment using any transport (agent),
 
 RabbitMQ stores metrics within a metrics-dedicated, non-durable, non-exclusive topic exchange.
```
Comment (bmansheim) [inline content/manager_architecture/metrics-flow.md]: RabbitMQ stores metrics within a metrics-dedicated, non-durable, non-exclusive topic exchange. After a metric is consumed it is removed from the queue.
If you remove the proprietary consumer, you can connect an external system to the RabbitMQ to consume metrics, but this is not an officially supported implementation.

File: content/manager_architecture/metrics-flow.md [hunk 2]
```diff
@@ -21,30 +21,40 @@ You can send metrics to the management environment using any transport (agent),
...
 
-After a metric is consumed it is removed from the queue. In principle, you can consume metrics directly from RabbitMQ for processing in systems outside Cloudify. Cloudify does not provide any implementation to officially support this, however it is enables by the architecture, and by removing the proprietary consumer, you it is possible to consume metrics directly from RabbitMQ.
+After a metric is consumed it is removed from the queue. In principle, you can consume metrics directly from RabbitMQ for processing in systems outside Cloudify. Cloudify does not provide any implementation to officially support this, however it is enables by the architecture, and by removing the proprietary consumer, it is possible to consume metrics directly from RabbitMQ.
+
+RabbitMQ connection is ssl secured and each deployments has its own vhost, hence connection parameters are the following:
```
Comment (bmansheim) [inline content/manager_architecture/metrics-flow.md]: The RabbitMQ connection is secured by SSL and each deployments has its own vhost. The connection parameters are:

File: content/manager_architecture/metrics-flow.md [hunk 3]
```diff
@@ -21,30 +21,40 @@ You can send metrics to the management environment using any transport (agent),
...
+password:	<by default: c10udify>
+port: <by default: 5672>
+exchange:	cloudify-monitoring
+routing-key:	*
+virtual_host: <deployment vhost>
+ssl_options:
+  ssl_enabled: True
+  cert_path: <path to internal certificate on manager it is under /etc/cloudify/ssl>
+```
+RabbitMQ credentials can be set on installation.
```
Comment (bmansheim) [inline content/manager_architecture/metrics-flow.md]: [installation]({{< relref "installation/installing-manager.md" >}}).

File: content/manager_policies/overview.md [hunk 4]
```diff
@@ -34,6 +34,8 @@ The workflow might increase the number of node instances, for example, to handle
 
 The logic fo the policies is written in [Clojure](http://clojure.org/) using Riemann's API. Cloudify adds a thin API layer on top of these APIs.
 
+A great example how to apply Cloudify policies can be found in [Nodecellar Example](https://github.com/cloudify-examples/nodecellar-auto-scale-auto-heal-blueprint)
```
Comment (bmansheim) [inline content/manager_policies/overview.md]: The [Nodecellar example](https://github.com/cloudify-examples/nodecellar-auto-scale-auto-heal-blueprint) is a great example how to apply Cloudify policies.

File: content/manager_architecture/metrics-flow.md (hunk 1 above)
Comment (szpotona) [reply content/manager_architecture/metrics-flow.md]: fixed

**Pull request merged**